### PR TITLE
TA-1430 Tests implementation

### PR DESCRIPTION
**Pull request by EvgeniiNikonets**
TA-1430 Tests implementation

### CODE REVIEW

File: test_files/rate_limit_active_checkbox_functional_tests.json [hunk 1]
```diff
@@ -0,0 +1,1311 @@
...
+          "Expected": {
+            "Code": "200",
+            "Type": "content",
+            "Single": true,
+            "Text": "b''"
+          }
+        },
+        {
+          "Name": "Delete Rate limit ",
+          "Skip": false,
```
Comment (olgasobolevsky) [inline test_files/rate_limit_active_checkbox_functional_tests.json:1289]: There is no need to delete rate limit at the end. It should be deleted by 'clean up' at the end of suite execution.

File: test_files/rate_limit_active_checkbox_functional_tests.json [hunk 2]
```diff
@@ -0,0 +1,1311 @@
...
+            "Text": "b''"
+          }
+        },
+        {
+          "ID": "wait-for-logs",
+          "Skip": false
+        },
+        {
+          "Name": "Assert logs",
+          "Skip": false,
```
Comment (olgasobolevsky) [inline test_files/rate_limit_active_checkbox_functional_tests.json:855]: Please specify the check for each "Assert logs" step. For example: "Assert logs - verify tag limit-name:test-ban2"

File: test_files/rate_limit_active_checkbox_functional_tests.json [hunk 3]
```diff
@@ -0,0 +1,1311 @@
...
+                "Value": "limit-name:active-rl"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "Name": "Rate limit ban - with 'Active' global filter",
+      "Skip": true,
```
Comment (olgasobolevsky) [inline test_files/rate_limit_active_checkbox_functional_tests.json:1025]: Change name: "Rate limit ban - with Global:active". ("global filter" - it's another security element in our system.)

File: test_files/rate_limit_active_checkbox_functional_tests.json [hunk 4]
```diff
@@ -0,0 +1,1290 @@
+{
+  "Name": "Rate limit - 'Active' checkbox functional tests TA-1430",
+  "Skip": false,
+  "Tests": [
+    {
+      "Name": "Rate limit - with 'Active' global filter",
+      "Skip": false,
```
Comment (olgasobolevsky) [inline test_files/rate_limit_active_checkbox_functional_tests.json:7]: Please, change name: "Rate limit - with 'Active' global filter" to "Rate limit - test 'Active' with 'Global':false".
('global filter' is one of the security components in the system and it is very confusing here)

File: test_files/rate_limit_active_checkbox_functional_tests.json (hunk 1 above)
Comment (EvgeniiNikonets) [reply test_files/rate_limit_active_checkbox_functional_tests.json]: Done

File: test_files/rate_limit_active_checkbox_functional_tests.json (hunk 3 above)
Comment (EvgeniiNikonets) [reply test_files/rate_limit_active_checkbox_functional_tests.json:1025]: Done

File: test_files/rate_limit_active_checkbox_functional_tests.json (hunk 4 above)
Comment (EvgeniiNikonets) [reply test_files/rate_limit_active_checkbox_functional_tests.json:7]: Done

**Review by olgasobolevsky — APPROVED** [review]

**Comment by olgasobolevsky** [comment]
The suites should be in test_suites folder, not test_files folder

**Pull request merged**